Helper/Elasticsearch/ClientRequestRuntime.php

```diff
@@ -83,10 +85,10 @@ public function data(string $input)
 
     /**
      * @param string $input
-     *
-     * @return array|false|null false when multiple results
+     * @return Document|null
+     * @throws EnvironmentNotFoundException
```

Comment:
I would only keep the @throws line in the docblock. All other info is in the code.

Reply:
it's now returning a Document